cmd/cluster/aws/destroy.go

```diff
@@ -21,11 +21,15 @@ func NewDestroyCommand(opts *core.DestroyOptions) *cobra.Command {
 
 	opts.AWSPlatform = core.AWSPlatformDestroyOptions{
 		AWSCredentialsFile: "",
+		RoleArn:            "",
```

Comment:
`""` is the zero-value for a string, and these struct literals call out the places where we have meaningful defaults - so let's not put anything in there that is just the zero-value, as it takes away from the ability to grok where our flags have meaningful defaults